.travis.yml

```diff
@@ -8,8 +8,6 @@ before_install:
   - yarn --version
   - npm install -g ganache-cli@latest
   - ganache-cli --version
-before_script:
-  - ganache-cli --defaultBalanceEther 500000000 > /dev/null &
```

Comment:
Coverage now uses your `development` network settings, but needs to launch the ganache server itself because it hooks directly into the evm to track code execution. 

This line had to be removed to avoid a 'port already in use' error.

Reply:
This is very nice of you to assist with these fixes! Thanks!